environmental-footprint/environmental-footprint/additional-content/environmental-footprint-calculator-estimation.mdx

```diff
@@ -0,0 +1,80 @@
+---
+meta:
+  title: Understanding environmental footprint estimation
+  description: This page explains how to understand the environmental footprint estimation of Elastic Metal servers.
+content:
+  h1: Understanding environmental footprint estimation
+  paragraph: This page explains how to understand the environmental footprint estimation of Elastic Metal servers.
+tags: environmental-footprint impact environment datacentre
+dates:
+  validation: 2024-10-30
+  posted: 2024-10-30
+categories:
+  - environmental-footprint
+---
+
+When you [order an Elastic Metal server](/bare-metal/elastic-metal/how-to/create-server/), you can see the estimated impact of your Elastic Metal server of choice before you order. In this page, we explain how the estimation is measured.
+
+<Message type="note">
+  The estimation described on this page is based on the calculation methodology detailed in the [Environmental Footprint calculation breakdown](/environmental-footprint/environmental-footprint/additional-content/environmental-footprint-calculator) documentation page.
+</Message>
+
+## Availability Zone footprint estimation
+
+When you select your Elastic Metal server's Availability Zone, you can see the quantity of CO2e emitted per kW consumed for each zone.
```

Comment:
Should we explain that CO2e = CO2 equivalent (the first time it appears on the page)?

Reply:
```suggestion
When you select your Elastic Metal server's Availability Zone, you can see the quantity of CO2 equivalent (CO2e) emitted per kW consumed for each zone.
```